e2e/cypress/integration/plugins/marketplace/marketplace_spec.js

```diff
@@ -272,7 +272,7 @@ describe('Plugin Marketplace', () => {
             cy.get('.more-modal__list').scrollIntoView().should('be.visible');
         });
 
-        it('autofocus on search plugin input box', () => {
+        it('MM-T1947 autofocus on search plugin input box', () => {
```

Comment:
```suggestion
        it('MM-T2001 autofocus on search plugin input box', () => {
```

Reply:
@prapti both MM-T1947 and MM-T2001 look same to me, are these test cases duplicate ?